emsize.py

```diff
@@ -37,32 +37,47 @@ def error(text):
 
 
 def parse_args(argv):
-  parser = argparse.ArgumentParser(description=__doc__)
-  parser.add_argument('-format', '--format')
-  parser.add_argument('file')
-  args = parser.parse_args(argv)
-  return args.file
+  files = []
+  params = []
+  appendFormat = True
+  ret = 0
+  for arg in argv:
+    if (arg.startswith('-')):
+      params.append(arg)
+    else:
+      files.append(arg)
+    if (arg.startswith('--format')):
+      appendFormat = False
+  if appendFormat:
+    params.append('--format=sysv')
+  for file in files:
+    ret = print_sizes(file, params)
+  return ret
+
+
+def print_sizes(file, params):
+  if not os.path.isfile(file):
+    return error('Input file %s not foune' % file)
+  ftype, fencoding = mimetypes.guess_type(file)
+  if (ftype == 'application/wasm' or ftype == 'text/vnd.a'):
```

Comment:
What what `guess_type` report as the type for a wasm object file? Object files should go into this path too.